R/Design-class.R

```diff
@@ -885,15 +885,26 @@ DADesign <- function(model, data,
 #' @slot first_cohort_mono_only (`flag`)\cr whether first test one mono agent cohort, and then
 #'   once its DLT data has been collected, we proceed from the second cohort onwards with
 #'   concurrent mono and combo cohorts.
-#' @slot same_dose (`flag`)\cr whether the lower dose of the separately determined mono and combo
+#' @slot same_dose_for_all (`flag`)\cr whether the lower dose of the separately determined mono and combo
 #'   doses should be used as the next dose for both mono and combo.
```

Comment:
Would "should be used as the next dose for both mono and combo _in all cohorts_" be clearer?

Reply:
yes